Pod/Classes/PINRemoteImageManager.m

```diff
@@ -999,29 +1000,21 @@ - (void)cancelTaskWithUUID:(NSUUID *)UUID
     __weak typeof(self) weakSelf = self;
     [_concurrentOperationQueue pin_addOperationWithQueuePriority:PINRemoteImageManagerPriorityHigh block:^
      {
-        typeof(self) strongSelf = weakSelf;
-        //find the task associated with the UUID. This might be spead up by storing a mapping of UUIDs to tasks
-        [strongSelf lock];
-            __block PINRemoteImageTask *taskToEvaluate = nil;
-            __block NSString *taskKey = nil;
-            [strongSelf.tasks enumerateKeysAndObjectsUsingBlock:^(NSString *key, PINRemoteImageTask *task, BOOL *stop) {
-                if (task.callbackBlocks[UUID]) {
-                    taskToEvaluate = task;
-                    taskKey = key;
-                    *stop = YES;
-                }
-            }];
-        
-            if (taskToEvaluate == nil) {
-                //maybe task hasn't been added to task list yet, add it to canceled tasks.
-                //there's no need to ever remove a UUID from canceledTasks because it is weak.
-                [strongSelf.canceledTasks addObject:UUID];
-            }
-        
-            if ([taskToEvaluate cancelWithUUID:UUID manager:strongSelf]) {
-                [strongSelf.tasks removeObjectForKey:taskKey];
-            }
-        [strongSelf unlock];
+         typeof(self) strongSelf = weakSelf;
+         [strongSelf lock];
+             NSString *taskKey = nil;
+             PINRemoteImageTask *taskToEvaluate = [strongSelf _locked_taskForUUID:UUID key:&taskKey];
+
+         if (taskToEvaluate == nil) {
```

Comment:
Indentation looks off here. Should this be indented within the -lock/-unlock? Or is GitHub just making it hard to see?